lib/src/screens/initial_setup/widgets/enc_key_input_form.dart

```diff
@@ -37,132 +39,155 @@ import 'package:form_builder_validators/form_builder_validators.dart';
 
 import 'package:solidpod/src/screens/initial_setup/initial_setup_constants.dart';
 
-/// A form widget for inputing encryption key
-///
-/// A developer may change this to include any other input data they want to
-/// gather from a POD user. Eg: Name, Gender
```

Comment:
If the comment is outdated, please update it appropriately rather than removing it.